tests/web/test_validators.py

```diff
@@ -0,0 +1,47 @@
+import pytest
```

Comment:
Do we have any tests for an actual MongoDB instance running and querying for `validators` collection?

Reply:
We don't store the validators in MongoDB, we [query tendermint directly](https://github.com/bigchaindb/bigchaindb/pull/2106/files#diff-c6511560546a7dc577e7e647b5bfdaceR248).